recipes/sdl/all/conandata.yml

```diff
@@ -2,7 +2,12 @@ sources:
   "2.0.14":
     url: "https://www.libsdl.org/release/SDL2-2.0.14.tar.gz"
     sha256: d8215b571a581be1332d2106f8036fcb03d12a70bae01e20f424976d275432bc
+  "2.0.16":
+    url: "https://www.libsdl.org/release/SDL2-2.0.16.tar.gz"
+    sha256: 65be9ff6004034b5b2ce9927b5a4db1814930f169c4b2dae0a1e4697075f287b
 patches:
   "2.0.14":
     - patch_file: "patches/0001-fix-cmake-ios-tvos.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/0002-mingw-improvements.patch"
+      base_path: "source_subfolder"
```

Comment:
sdl2-config file is not relocatable and removed from packaging, so I don't think that this patch is required, unless if there was more logic.

Reply:
not sure if it's good idea... I've seen some projects looked for the `sdl2-config` as sdl2 was their dependency (don't remember exact names, but something mpeg related).
maybe we should make `sdl2-config` relocatable instead (or generate one, like we did with freetype)

Reply:
It defeats CCI policy

Reply:
Perhaps conan needs a generator for creating these config scripts.
These often have a common structure.